packages/salesforcedx-apex-replay-debugger/package.json

```diff
@@ -13,6 +13,7 @@
   "categories": ["Debuggers"],
   "dependencies": {
     "@salesforce/salesforcedx-utils-vscode": "43.4.0",
+    "request-light": "0.2.1",
```

Comment:
Added for XHROptions and XHRResponse required for testing.

Reply:
If this is only for testing does it need to be in dependencies? Or can it go into devDependencies?

Reply:
I put it in the same place it was used for the vscode-apex-debugger extension

Reply:
apex-debugger used to have its REST services in that package, so request-light was a dependency. That stuff got moved to utils (https://github.com/forcedotcom/salesforcedx-vscode/pull/389) and all that's left in apex-debugger that needs request-light are tests. Changing the dependency probably just got missed.

You should try changing this to devDependency.